test/unit/content-info.test.js

```diff
@@ -0,0 +1,435 @@
+var expect = require('expect.js'),
+    contentInfo = require('../../lib/content-info/index'),
+    Response = require('../../lib/index.js').Response;
+
+describe('contentInfo module', function () {
+    it('Should take the content-type from the response stream if content-type headers is not present', function () {
+        // data url of png image
+        var img = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAYAAACNiR0' +
+        'NAAAAKElEQVQ4jWNgYGD4Twzu6FhFFGYYNXDUwGFpIAk2E4dHDRw1cDgaCAASFOffhEIO' +
+        '3gAAAABJRU5ErkJggg==',
+            // replacing the mime type and encoded format
+            data = img.replace(/^data:image\/\w+;base64,/, ''),
```

Comment:
`img` isn't being used elsewhere in this test, why not use hardcoded `data` directly?

Reply:
It is good to have it this way to improve readability. Let's not DRY the tests too much.